### PR TITLE
Add breaking change for liveness_probe path and port deprecation

### PR DESCRIPTION
**Pull request by mtgoncalves1**
Related issue: https://gitlab.com/architect-io/architect-cli/-/issues/649.

Add release note section for liveness_probe `path` and `port` deprecation.

### CODE REVIEW

File: reference/release-notes.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ Full changelog can be viewed on our [GitHub Repo](https://github.com/architect-t
 
 <br/>
 
-# Breaking Change: Replace parameters with secrets keyword
+# Breaking Change #1: Replace parameters with secrets keyword
```
Comment (Zach-at-Arch) [inline reference/release-notes.mdx:13]: For the breaking change, two things we want to do to help categorize them:

1. Keep h1's to the month the change will hit.  So the secrets/parameters would be # Feb 2023, and this new one would be # April 2023 (assuming we get the change in before may)
2. Everything else should be H2's (##) so that the content of each month will be categorized nicely in the right side bar

File: reference/release-notes.mdx [hunk 2]
```diff
@@ -116,3 +116,34 @@ or (the non-shorthand version):
 ```
 [...] --secret key=value [...]
 ```
+
+# Breaking Change #2: Deprecation `path` and `port` for `liveness_probe`
+
+Starting from October of 2023, the usage of `path` and `port` for the `liveness_probe` will no longer be supported. Instead, the `command` option should be used for defining the liveness probe.
```
Comment (Zach-at-Arch) [inline reference/release-notes.mdx:122]: @mtgoncalves1 are we planning on releasing this change in Oct?

File: reference/release-notes.mdx [hunk 3]
```diff
@@ -116,3 +116,34 @@ or (the non-shorthand version):
...
+
+## Who does this change affect?
+This change affects all users who are currently using the `path` and `port` options in their `liveness_probe` configuration. If you are one of them, please update your configuration to use the `command` option before October to avoid any disruptions in your application's health checks.
+
+## Why are we making this change?
+We are making this change to favor the `command` option, as it provides more flexibility in defining the liveness probe command.
```
Comment (Zach-at-Arch) [inline reference/release-notes.mdx:128]: Mention some of the errors we've ran into regarding the liveness probe command here with path/port

**Review by Zach-at-Arch — CHANGES_REQUESTED** [review]
@mtgoncalves1 I have some general overhaul to how the release notes are structured; once you change that I'll give it another pass through

File: reference/release-notes.mdx (hunk 2 above)
Comment (mtgoncalves1) [reply reference/release-notes.mdx:122]: I'm not sure when should we release it. I just put `October` as a placeholder. What do you think?

File: reference/release-notes.mdx [hunk 4]
```diff
@@ -10,7 +10,46 @@ Full changelog can be viewed on our [GitHub Repo](https://github.com/architect-t
...
+## Examples
+Example: if you were using `path` and `port` in your component config as follow:
+```
+services:
+  app:
+    liveness_probe:
+      port: 3000
+      path: /
+```
+You would change this to a `curl` command, or a `wget` command. Whichever command you decide to use, make sure that it exists in your image.
```
Comment (Zach-at-Arch) [inline reference/release-notes.mdx:38]: "You would change this to `curl` or `wget`, depending on which is installed on your image."

File: reference/release-notes.mdx [hunk 5]
```diff
@@ -10,7 +10,46 @@ Full changelog can be viewed on our [GitHub Repo](https://github.com/architect-t
 
 <br/>
 
-# Breaking Change: Replace parameters with secrets keyword
+# April 2023 Breaking Change: Deprecation `path` and `port` for `liveness_probe`
```
Comment (Zach-at-Arch) [inline reference/release-notes.mdx:13]: Split this up into two parts:
# April 2023

## Breaking Change: Depreciation ....

Also, for any headers below make sure to switch from # to ### ( so this would be the "who does this change affect? Why are we making this change? etc.)

Make sure you do the same thing for the February change as well.

TL;DR on why - The headings determine how the sidebar looks - right now it's not super helpful 
image

The update should make it look more like this:

- April 2023
  - Breaking change ..
    - Who does this change affect?
    - Why are we making this change?
    - etc....
- February 2023
  - Breaking change ..
    - Who does this change affect?
    - Why are we making this change?
    - etc....

**Review by Zach-at-Arch — CHANGES_REQUESTED** [review]
Few structural and grammatical changes. Should be good after that

**Review by Zach-at-Arch — APPROVED** [review]
